interview-questions.md

```diff
@@ -8,36 +8,53 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: Any missing information from he database like a missing foreign key or columns can be fixed by perfoming a migration. The foreign key would be stored in the Cohort model and the name of the foreign key would be cohort_id.
 
-Researched answer:
+Researched answer: Since we are dealing with Active Records Associations we are establishing a relationship between to seperate tables in the database. Active Records Asscoiation uses the concept of has_many and belongs_to in order to establish this relationship. In this scenario, a Cohort has_many students which means that Students belong_to a cohort. To add a column for the foreign key you would do the following:
+rails generator or g migration add_foreign_key
```

Comment:
great explanation, just one mistake - you would add the foreign key to the belongs_to side, which is Student.